core/templates/pages/contributor-dashboard-page/translation-language-selector/translation-language-selector.component.html

```diff
@@ -1,4 +1,4 @@
-<div #dropdown class="oppia-translation-language-selector-container e2e-test-language-selector">
+<div #dropdown tabindex="1" class="oppia-translation-language-selector-container e2e-test-language-selector">
   <div class="oppia-translation-language-selector-featured-tooltip">
```

Comment:
@akshayjoshi892 any specific reason to add `tabindex` ?

Reply:
@akshayjoshi892 The correct way to make a PR is mentioned [here](https://github.com/oppia/oppia/wiki/Make-a-pull-request), you can have a look.
One PR should include the changes that solves a particular issue only, i.e. you should always use a separate branch for a different PR.

Thanks.

Reply:
I got it @kshitij01042002 ,I will make changes thanks!